chapters/unitexpressions.tex

```diff
@@ -73,6 +74,8 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
 
 \begin{nonnormative}
 A tool may present \lstinline!Ohm! as $\Omega$ and the prefix \lstinline!u! as $\mu$, and similarly \lstinline!m2! as m\textsuperscript{2}.
+In addition a tool may present \lstinline!deg! as $^{\circ}$ (for angle) and \lstinline!degC! as $^{\circ}$C (for temperature).
```

Comment:
This isn't generic enough to cover `"degF"`.

Reply:
I don't see that it is needed as it is non-normative, and I find the next line more important and that one is only relevant for the SI-standard, and thus not for Fahrenheit. Explaining that would be too much text.

Reply:
But I tried to add a bit more.

Reply:
Looks good now.  Resolving.